app/views/devise/confirmations/sent.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :title, t("confirmation_sent.heading") %>
-<% service = service_for(params[:previous_url], current_user) %>
+<% service = @user_is_new ? service_for(params[:previous_url], current_user) : nil %>
```

Comment:
Could we shorten it even beyond the ternery?
```
<% service = service_for(params[:previous_url] if @user_is_new %>
```

That should also make service == nil if @user_is_new is falsy?

Reply:
there's also rightward assignment that I've not used yet... Not sure what our style guide is for that yet. But i think that's from 2.7.2 onwards?

Reply:
I think that would be:

```
<% service_for(params[:previous_url] if @user_is_new => service %>
```

But haven't tried it and am on the fence if that's any clearer